setup.py

```diff
@@ -19,6 +19,7 @@ def read_file(file):
       license='MIT',
       zip_safe=False,
       keywords='gis geospatial geographic shapefile shapefiles',
+      python_requires='>= 2.7, >= 3',
       classifiers=['Programming Language :: Python',
                    'Topic :: Scientific/Engineering :: GIS',
                    'Topic :: Software Development :: Libraries',
```

Comment:
these classifiers can also be used to specify python version (i guess it only shows up on the pypi page though and is not used programmatically by e.g. pip, dunno)

https://github.com/obspy/obspy/blob/3ffa3209c637026dc922540f68ad31c9cf69780a/setup.py#L784-L790